src/components/TodoList/TodoList.tsx

```diff
@@ -16,85 +25,41 @@ export const TodoList: React.FC = () => (
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-      <tr data-cy="todo" className="has-background-info-light">
-        <td className="is-vcentered">2</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">quis ut nam facilis et officia qui</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye-slash" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">6</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">
-            qui ullam ratione quibusdam voluptatem quia omnis
-          </p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
+      {todos.map(todo => (
+        <tr
+          key={todo.id}
+          data-cy="todo"
+        >
+        <td className="is-vcentered">{todo.id}</td>
         <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
+        {todo.completed && (
+           <span className="icon" data-cy="iconCompleted">
+              <i className="fas fa-check" />
           </span>
+        )}
         </td>
         <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
+          <p className={todo.completed ? 'has-text-success' : 'has-text-danger'}>
+            {todo.title}
+          </p>
         </td>
         <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
+          <button
+           data-cy="selectButton"
+           className="button"
+           type="button"
+           onClick={() => openModal(todo.id)}
```

Comment:
```suggestion
           onClick={() => setSelectedTodo(todo)}
```
we can save in that state the whole todo